SourceryTests/Models/TypeNameSpec.swift

```diff
@@ -111,6 +111,9 @@ class TypeNameSpec: QuickSpec {
                     expect(TypeName("() -> (Int, Int)").isClosure).to(beTrue())
                     expect(TypeName("() -> (Int) -> (Int)").isClosure).to(beTrue())
                     expect(TypeName("((Int) -> (Int)) -> ()").isClosure).to(beTrue())
+                    expect(TypeName("(Foo<String>) -> Bool)").isClosure).to(beTrue())
+                    expect(TypeName("(Int) -> Foo<Bool>").isClosure).to(beTrue())
+                    expect(TypeName("(Foo<String>) -> Foo<Bool>)").isClosure).to(beTrue())
```

Comment:
nit: this does not seem to be a valid type actually, the last ) is extra, did you mean it as a single value tuple?

Reply:
thanks for pointing out, it was a typo, fixed now.